docs/build/cheerio-scraper-tutorial.md

```diff
@@ -138,13 +138,17 @@ const { url } = request;
 
 // ...
 
-const uniqueIdentifier = url.split('/').slice(-2).join('/');
+const uniqueIdentifier = url
+    .split('/')
+    .slice(-2)
+    .join('/');
 
 return {
     url,
     uniqueIdentifier,
     title: $('header h1').text(),
-    description: $('header p[class^=Text__Paragraph]').text(),
+    description: $('header p[class^=Text__Paragraph]')
+        .text(),
```

Comment:
Not sure if this makes it more readable. One could easily miss the method invocation. Maybe save the selector into a short-named variable instead?

This is repeated multiple times so please change it everywhere.